pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,6 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.36.0-dev34"
+
+__version__ = "0.36.0-dev35"
+
```

Comment:
Since the `setup.py` reads the last line of this file, the added whitespace was causing the version to be wrong, causing the setup to fail.
```suggestion
__version__ = "0.36.0-dev35"
```